Documentation/runtime-configuration.md

```diff
@@ -8,40 +8,37 @@ Reconfiguration requests can only be processed when the the majority of the clus
 
-[member migration]: https://github.com/coreos/etcd/blob/master/Documentation/admin_guide.md#member-migration
+[remove member]: #remove-a-member
+[add member]: #add-a-new-member
+[member migration]: #member-migration
 
-### Increase Cluster Size
+### Cycle or Upgrade Multiple Machines
 
-To make your cluster more resilient to machine failure you can increase the size of the cluster.
-For example, if the cluster consists of three machines, it can tolerate one failure.
-If we increase the cluster size to five, it can tolerate two machine failures.
+If you need to move multiple members of your cluster due to planned maintenance (hardware upgrades, network downtime, etc.), it is recommended to complete the process member by member, instead of all at once. It is safe to remove the leader, however there is a brief period of downtime while the election process takes place.
 
-Increasing the cluster size can also provide better read performance.
-When a client accesses etcd, the normal read gets the data from the local copy of each member (members always shares the same view of the cluster at the same index, which is guaranteed by the sequential consistency of etcd).
-Since clients can read from any member, increasing the number of members thus increases overall read throughput.
 
-### Decrease Cluster Size
+### Change the Cluster Size
 
-To improve the write performance of a cluster, you might want to trade off resilience by removing members.
-etcd replicates the data to the majority of members of the cluster before committing the write.
-Decreasing the cluster size means the etcd cluster has to do less work for each write, thus increasing the write performance.
+Increasing the cluster size can enhance [failure tolerance][fault tolerance table] and provide better read performance. Since clients can read from any member, increasing the number of members increases the overall read throughput.
+
+Decreasing the cluster size can improve the write performance of a cluster, with a trade-off of decreased resilience. Writes into the cluster are replicated to a majority of members of the cluster before considered committed. Decreasing the cluster size lowers the majority, and each write is committed more quickly.
+
+[fault tolerance table]: admin_guide.md#fault-tolerance-table
 
 ### Restart Cluster from Majority Failure
 
 If the majority of your cluster is lost, then you need to take manual action in order to recover safely.
-The basic steps in the recovery process include creating a new cluster using the old data, forcing a single member to act as the leader, and finally using runtime configuration to add members to this new cluster.
+The basic steps in the recovery process include creating a new cluster using the old data, forcing a single member to act as the leader, and finally using runtime configuration to [add new members][add member] to this new cluster one at a time.
```

Comment:
can you please add a link to https://github.com/coreos/etcd/blob/master/Documentation/admin_guide.md#disaster-recovery?